docs/getting_started/basic/zero-shot-prediction/zero-shot-detection.ipynb

```diff
@@ -0,0 +1,1378 @@
+{
```

Comment:
<p>Opening sentence:</p><p>Now that you understand the basics of zero-shot models for classification[link to your previous notebook], let's look at using them for Object Detection[link to something]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>